app/routes/application.js

```diff
@@ -2,7 +2,7 @@ import Route from '@ember/routing/route';
 import { get } from '@ember/object';
 import { inject as service } from '@ember/service';
 
-const DEV = false;
+const DEV = true;
```

Comment:
what does changing this do?

Reply:
oops, should be `false`, it simulates the contentful environent for local dev